### PR TITLE
Layout/landing page - #27

### PR DESCRIPTION
**Pull request by sara219**
Create the landing page and add : 
- [x] header (logo and menu)
- [x]  Picture 
- [x]  Call to action to go to questionnaire 
- [x]  link or button to resources

### CODE REVIEW

File: src/CommonComponents/Header/index.style.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ width: 40rem;
 `;
 
 export const Menu = styled.img`
+color : red;
```
Comment (hshahwan) [inline src/CommonComponents/Header/index.style.js]: no need for this color

Comment (sara219) [reply src/CommonComponents/Header/index.style.js]: @hshahwan 
I delete it!

**Review by hshahwan — APPROVED** [review]

**Review by someyaaltous — APPROVED** [review]

**Review by SajaLahaleeh — APPROVED** [review]

File: src/Layout/LandingPage/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  setRender = () => {
+    this.setState({
+      renderLink: true
+    })
+  }
+
+  renderLink = () => {
+    if (this.state.renderLink) {
+      return <Redirect to='/resource' />
```
Comment (95Rawan) [inline src/Layout/LandingPage/index.js]: The endpoint name should be resources not resource

File: src/Layout/LandingPage/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom'
+import { Title, LandingImg, Paragraph, Button, ResourceLink, Link } from './index.style';
+import landing from '../../assets/landing-img.jpeg';
+import Header from '../../CommonComponents/Header';
```
Comment (95Rawan) [inline src/Layout/LandingPage/index.js]: We should only call the header component in the app because it shows in almost all of the pages

**Review by 95Rawan — APPROVED** [review]

File: src/Layout/LandingPage/index.js (hunk 2 above)
Comment (sara219) [reply src/Layout/LandingPage/index.js]: @95Rawan 
I Fixed it!

File: src/Layout/LandingPage/index.js (hunk 3 above)
Comment (sara219) [reply src/Layout/LandingPage/index.js]: @95Rawan  
Done!

**Pull request merged**